src/mmw/js/src/modeling/templates/projectMenu.html

```diff
@@ -8,7 +8,7 @@ <h3 class="light">{{ name }}</h3>
         <i class="fa fa-caret-down"></i>
         </button>
         <ul class="dropdown-menu menu-right" role="menu">
-            {% if id and editable %}
+            {% if id and not isNew %}
```

Comment:
Checking both is redundant since they will always have exactly the same value. Consider using just `if id` or `if not isNew`